scaleway/resource_rdb_user_test.go

```diff
@@ -19,7 +19,7 @@ func init() {
 func TestAccScalewayRdbUser_Basic(t *testing.T) {
 	tt := NewTestTools(t)
 	defer tt.Cleanup()
-	instanceName := "TestAccScalewayRdbUser_Basic"
```

Comment:
Why this rename is needed?

Reply:
I was bizarre a camel case naming in the console